api/src/main/resources/liquibase-update-to-latest.xml

```diff
@@ -6337,6 +6337,11 @@
 	</changeSet>
 	
 	<changeSet id="20111222-1659" author="djazayeri">
+		<preConditions onFail="MARK_RAN">
```

Comment:
It is good to have preconditions, but we try to avoid editing changesets, especially one like this where we don't expect one to have the table since it is new, have you tried to investigate why it already exists because that's what you might want to fix instead of this.